docs/products/clickhouse/get-started.md

```diff
@@ -5,77 +5,498 @@ sidebar_label: Get started
+
+      ```hcl
+      resource "aiven_clickhouse" "clickhouse" {
+        project      = var.project_name
+        service_name = var.service_name
+        cloud_name   = var.cloud_name
+        plan         = var.service_plan
+      }
+
+      output "clickhouse_service_host" {
+        value = aiven_clickhouse.clickhouse.service_host
+      }
+
+      output "clickhouse_service_port" {
+        value = aiven_clickhouse.clickhouse.service_port
+      }
+
+      output "clickhouse_service_username" {
+        value = aiven_clickhouse.clickhouse.service_username
+      }
+
+      output "clickhouse_service_password" {
+        value     = aiven_clickhouse.clickhouse.service_password
+        sensitive = true
+      }
+      ```
+
+   - `variables.tf` for declaring your project variables
+
+      ```hcl
+      variable "aiven_api_token" {
+        description = "Aiven API token"
+        type        = string
+      }
+
+      variable "project_name" {
+        description = "Project name"
+        type        = string
+      }
+
+      variable "cloud_name" {
+        description = "Cloud name"
+        type        = string
+      }
+
+      variable "service_name" {
+        description = "Service name"
+        type        = string
+      }
+
+      variable "service_plan" {
+        description = "Service plan"
+        type        = string
+      }
+      ```
+
+   - `terraform.tfvars` for assigning actual values to your previously declared variables
+
+      ```hcl
+      project_name = "testproject-o3jb"
+      cloud_name   = "google-europe-west1"
+      service_name = "clickhouse"
+      service_plan = "startup-16"
+      ```
+
+1. Run `terraform init` > `terraform plan` > `terraform apply --auto-approve`.
+
+1. Store Terraform outputs in environment variables so that they can be used for
+   [connecting](#connect-to-service):
+
+   ```bash
+   CLICKHOUSE_HOST="$(terraform output -raw clickhouse_service_host)"
+   CLICKHOUSE_PORT="$(terraform output -raw clickhouse_service_port)"
+   CLICKHOUSE_USER="$(terraform output -raw clickhouse_service_username)"
+   CLICKHOUSE_PASSWORD="$(terraform output -raw clickhouse_service_password)"
+   ```
+
+</TabItem>
+<TabItem value="3" label="K8s">
+
+Create an Aiven for ClickHouse service using
+[Aiven Operator for Kubernetes](https://aiven.github.io/aiven-operator/api-reference/clickhouse.html).
+
+1. Create file `example.yaml` with the following content:
+
+   ```yaml
+   apiVersion: aiven.io/v1alpha1
+   kind: Clickhouse
+   metadata:
+     name: my-clickhouse
+   spec:
+     authSecretRef:
+       name: aiven-token
+       key: token
+
+     connInfoSecretTarget:
+       name: my-clickhouse-connection
+       annotations:
+         foo: bar
+       labels:
+         baz: egg
+
+     tags:
+       env: test
+       instance: foo
+
+     userConfig:
+       ip_filter:
+         - network: 0.0.0.0/32
+           description: bar
+         - network: 10.20.0.0/16
```

Comment:
I wouldn't use ip filters in this guide. While it might demonstrate the possibility in "api reference" this will actually change service's availability.